src/spec-configuration/httpOCIRegistry.ts

```diff
@@ -249,16 +193,105 @@ async function existsInPath(filename: string): Promise<boolean> {
 	if (!process.env.PATH) {
 		return false;
 	}
-	const paths = process.env.PATH.split(':');
-	for (const path of paths) {
-		const fullPath = `${path}/${filename}`;
-		if (await isLocalFile(fullPath)) {
-			return true;
+	try {
+		const paths = process.env.PATH.split(':');
+		for (const path of paths) {
+			const fullPath = `${path}/${filename}`;
+			if (await isLocalFile(fullPath)) {
+				return true;
+			}
 		}
+	} catch (err) {
```

Comment:
When is an error thrown? (Wondering if we might hide other issues with the catch-all.)

Reply:
Looking again, I don't have a great reason in mind to keep this. Removing.